mitxpro/utils.py

```diff
@@ -485,6 +485,40 @@ def make_csv_http_response(*, csv_rows, filename):
     return response
 
 
+def make_csv_http_response_with_instructions(*, csv_rows, filename):
+    """
+    Create a HttpResponse for a CSV file.
```

Comment:
Please update the docstring to include "with instructions at the start of the file"